### DISCUSSION TITLE
network link light not coming on

### DISCUSSION THREAD

**Discussion by pintail120** [WLAN-Pi/feedback#54] [General Inquiries]
The wlan pro is booted up, however....
When I plug it into any switch the network link led does not come on, the PI has one constant orange light on the network port but the other led remains off no matter what I plug it into ?
any ideas why the network port is not turning on ?

**Comment by jiribrejcha** [comment]
I see you have already tried multiple switches, and I assume also a different Ethernet cable?

Connect the Pro to a switch:
1. Do you see any unusual message in output when you SSH and execute the `dmesg` command? Feel free to share the output here.
2. Can you share the output of this command `sudo ethtool eth0`?
3. What is the output of this command `sudo ip link show`?
4. What does `sudo ethtool -i` say?

Have you tried reflashing the SD card with a fresh software image?
